pint/models/priors.py

```diff
@@ -0,0 +1,72 @@
+"""priors.py
+Defines classes used for evaluation prior probabilities
+
+Initially this handles priors on single parameters that don't depend on any
+other parameters.  This will need to be supplemented with a mechanism, probably
+in the model class, that implements priors on combinations of parameters,
+such as total proper motion, 2-d sky location, etc.
+
+"""
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+
+class Prior(object):
+    r"""Class for evaluation of prior probability densities
+    
+     Any Prior object returns the probability density using
+    the prior_probability() method.  For generality, these 
+    are written so that they work on a scalar value or a numpy array
+    of values.
+    
+    The default Prior is just the constant value 1.0.
+    Subclasses implement other forms.
+   
+    """
+    
+    def __init__(self):
+        pass
+    
+    def prior_probability(self,value):
+        return np.ones_like(value)
+        
+
+class UniformBoundedPrior(Prior):
+    r"""A uniform prior between two bounds
+    
+    Parameters
+    ----------
+    lower_bound : number
+        Lower bound of allowed parameter range
+        
+    upper_bound : number
+        Upper bound of allowed parameter range
+    """
+    def __init__(self,lower_bound,upper_bound):
+        super(UniformBoundedPrior,self).__init__()
+        self.lower = lower_bound
+        self.upper = upper_bound
+                
+    def prior_probability(self,value):
+        return np.logical_and(value>self.lower,value<self.upper)*np.ones_like(value)
+
+class GaussianPrior(Prior):
+    def __init__(self, mean, sigma):
+        super(GaussianPrior,self).__init__()
+        self.mean = mean
+        self.sigma = sigma
+        self.var = sigma**2
+
+    def prior_probability(self,value):
+        return np.exp(-(value-self.mean)**2/(2*self.var))/np.sqrt(2.0*np.pi*self.var)
```

Comment:
This computation could have been done, perhaps more clearly and with less chance for error, by using scyipy.stats.norm(). Would that be encouraged or discouraged? Is it bad to have another dependency on scipy (obviously scipy is already used elsewhere in PINT, but I didn't know if we were trying to minimize dependencies like that).  What do you say @scottransom ?